cypress/integration/examples/network_requests.spec.js

```diff
@@ -87,6 +87,60 @@ context('Network Requests', () => {
     })
   })
 
+  it('cy.request() - pass result to the second request', () => {
+    // first, let's find out the userId of the first user we have
+    cy.request('https://jsonplaceholder.cypress.io/users?_limit=1')
+      .its('body.0') // yields the first element of the returned list
+      .then((user) => {
+        expect(user).property('id').to.be.a('number')
+        // make a new post on behalf of the user
+        cy.request('POST', 'https://jsonplaceholder.cypress.io/posts', {
+          userId: user.id,
+          title: 'Cypress Test Runner',
+          body: 'Fast, easy and reliable testing for anything that runs in a browser.',
+        })
+      })
+      // note that the value here is the returned value of the 2nd request
+      // which is the new post object
+      .then((response) => {
```

Comment:
I feel like this 'then' placement is a bit confusing since there was no `return` in the previous then....honestly, I'm not sure how this is working without a `return` statement. https://on.cypress.io/then#Yields

Reply:
the yielded result from the last command in `then` is yielded even if the command is NOT returned. This test passes just fine in 3.4.0
```js
    cy.wrap(3)
      .then((what) => {
        expect(what).to.equal(3)
        cy.then(() => 4)
      })
      .then((second) => {
        expect(second).to.equal(4)
      })
```

Reply:
ok, this is expected behavior for commands, we will update our docs https://github.com/cypress-io/cypress-documentation/issues/1902